rfm_dashboard/controller/landing_page.py

```diff
@@ -0,0 +1,9 @@
+from flask import Blueprint, render_template
+from ..models.get_rfm_data import get_rfm_data
+
+bp = Blueprint('landing_page', __name__)
+
+@bp.route('/rfm',methods=['GET','POST'])
```

Comment:
In the RFM page we want to access the page and expect the page to show the RFM data, then we need only GET HTTP methods krub. 

The POST HTTP method is the HTTP verb for submit create request krub.
ref: https://developer.mozilla.org/en-US/docs/Web/HTTP/Methods